gulpfile.js

```diff
@@ -235,7 +232,6 @@ gulp.task('dev', function() {
 
 gulp.task('default', function(cb) {
   runSequence(
-    'check-dependencies',
```

Comment:
this would break adding after adding the packaged fbjs in addition to having the normal one